feedback.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Feedback</title>
+</head>
+<body>
+<h2>
+  Contact Us
+</h2>
+
+<form name="feeedback" action="success.html" method="post">
+  <div>
+    <label>Feedback Type:</label>
+    <br>
+    <select name="feedback_type">
+      <option value="Suggestion or Request">Suggestion or Request</option>
+      <option value="Compliment">Compliment</option>
+      <option value="Constructive Criticism">Constructive Criticism</option>
+      <option value="Recipe Question">Recipe Question</option>
+    </select>
+  </div>
+
+  <div>
+    <label>Enter your message below:</label>
+    <br>
+    <textarea name="comments" rows="15" cols="70">
+      Start typing here...
```

Comment:
Good idea, unfortunately it looks like the placeholder works with the input tag, not the textarea tag.